TESTS/storage_abstraction/basicAPI/basicAPI.cpp

```diff
@@ -911,7 +916,7 @@ control_t test_programDataWithMultipleProgramUnits(const size_t call_count)
             TEST_ASSERT(rc >= 0);
             if (rc == ARM_DRIVER_OK) {
                 TEST_ASSERT_EQUAL(1, capabilities.asynchronous_ops);
-                return CaseTimeout(500);
+                return CaseTimeout(1000);
```

Comment:
This one was hitten too. I don't remember the N value but with this timeout the test pass.